### PR TITLE
Implement the basic functionality of stack using Array Data structure

### PR DESCRIPTION
**Pull request by Rishikeshrajrxl**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Yes.)

### Description

(Implemented the basic functionality of stack such as push, pop, peek,is_Empty, is_Full, without using Inbuilt Functions.)

### Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(#399 )

### CODE REVIEW

**Comment by Rishikeshrajrxl** [comment]
@siddharth25pandey 
Kindly review it and let me know any change needed.

**Comment by Rishikeshrajrxl** [comment]
@iamrajiv , @plazzy99  please review this.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@Rishikeshrajrxl fixed typos mistakes across the files. Also, remove unnecessary line breaks.

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@Rishikeshrajrxl Please make following modifications
* count function is missing, which gives total number of elements present at that time.
* Remove unnecessary line breaks & space

**Comment by Rishikeshrajrxl** [comment]
> @Rishikeshrajrxl fixed typos mistakes across the files. Also, remove unnecessary line breaks.

 @iamrajiv ,    What is fixed typos mistake across the file ? Please explain me.

File: C-Plus-Plus/ds/Stack_using_LinkedList.cpp [hunk 1]
```diff
@@ -1,3 +1,9 @@
+//This program implements Stack Data stucture using Linked List representation
+
+//by @GudlaArunKumar
+
+
+
 // Stsck using Vector
```
Comment (iamrajiv) [inline C-Plus-Plus/ds/Stack_using_LinkedList.cpp]: remove Stsck

File: C-Plus-Plus/ds/Stack_using_Array.cpp [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+Implement the basic functionality of stack using Array Data structure
+
+@author: Rishikeshrajrxl
+@created: 17/08/2020
+
+*/
+
+#include <iostream>
+using namespace std;
+
```
Comment (iamrajiv) [inline C-Plus-Plus/ds/Stack_using_Array.cpp:9]: remove the unnecessary line break

File: C-Plus-Plus/ds/Stack_using_Array.cpp [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+void Push();             //Function declaration
+void Peek();
+void Pop();
+void Is_Empty();
+void Is_Full();
+void Display();
+int stack_size,top=0,stack[100];
+
+
```
Comment (iamrajiv) [inline C-Plus-Plus/ds/Stack_using_Array.cpp:18]: remove the unnecessary line break

**Review by iamrajiv — CHANGES_REQUESTED** [review]

**Comment by Rishikeshrajrxl** [comment]
I have made all changes, please review this.

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@Rishikeshrajrxl , follow the same workflow for Linked list implementation, as what you have done for working with array. 
* work on user interface

**Comment by Rishikeshrajrxl** [comment]
@siddharth25pandey , Done with all changes.

**Review by siddharth25pandey — APPROVED** [review]
@Rishikeshrajrxl  Looks fine to me.
Able to merge
Thanks!

**Review by vatsalkesarwani12 — APPROVED** [review]
The code looks fine for me.

**Review by iamrajiv — APPROVED** [review]

**Comment by Rishikeshrajrxl** [comment]
@siddharth25pandey , This pull request is approved, please merge it.

**Pull request merged**